ForumProject/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.9"
+
+services:
+  postgres_lib:
+    image: postgres:latest
+    container_name: postgres_lib
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+
+  forum:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: forum
+    depends_on:
```

Comment:
Although `depends_on` ensures that `postgres_lib` starts before `forum`, it doesn't guarantee that PostgreSQL is ready to accept connections when the `forum` service starts. Consider using a tool like `wait-for-it` or `dockerize` to delay the forum service startup until the database is actually ready to accept connections

Reply:
Thanks, this has been implemented with 'dokerize' tool